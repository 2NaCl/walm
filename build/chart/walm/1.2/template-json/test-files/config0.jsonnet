{
  LICENSE_ADDRESS: '172.16.1.41:2181',
  Transwarp_Alias: 'walm_0000',
  Transwarp_App_Labels: {
    'transwarp.app': 'walm',
  },
  Transwarp_App_Name: 'walm',
  Transwarp_App_Scope: 'namespace',
  Transwarp_DEBUG: true,
  Transwarp_Install_ID: '1160',
  Transwarp_Install_Namespace: 'u5003',
  Transwarp_Namespace_Owner: 'flannel',
  Transwarp_Principal_Suffix: 'transwarp.local',
  Transwarp_Realm: 'TOS',
  Transwarp_Registry_Server: '172.16.1.41:5000',
  TosVersion: '1.5',

  App: {
    image: "172.16.1.99:5000/gold/walm:tdc-1.0",
    priority: 0,
    replicas: 1,
    resources: {
      cpu_limit: 4,
      cpu_request: 1,
      memory_limit: 4,
      memory_request: 2,
    }
  },

  
  Advance_Config: {
      debug: false,
      port: 8000,
      httpreadtimeout: 0,
      httpwritetimeout: 0,
      zipkinurl: "",
      tiller_connection_timeout: 0,
      kube_context: "default",
      oauth: false,  
      JwtSecret:"",
      
      dbname: "walm",
      dbtype: "mysql",
      dbtabpre: "",

    },

    MYSQL_CLIENT_CONFIG: {}, 
}
