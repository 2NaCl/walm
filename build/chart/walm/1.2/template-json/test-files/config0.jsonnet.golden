{
   "dummy-svc.json": {
      "apiVersion": "v1",
      "kind": "Service",
      "metadata": {
         "annotations": {
            "transwarp.meta": "{\"provides\": {\"WALM_CONFIG\": {\"immediate_value\": {\"walm_http_port\": 8000}}}}"
         },
         "labels": {
            "transwarp.alias": "walm_0000",
            "transwarp.app": "walm",
            "transwarp.install": "1160",
            "transwarp.meta": "true",
            "transwarp.name": "app-dummy",
            "transwarp.scope": "namespace",
            "transwarp.svc.scope": "app"
         },
         "name": "app-dummy-1160"
      },
      "spec": {
         "clusterIP": "None",
         "ports": [
            {
               "port": 5000,
               "targetPort": 5000
            }
         ],
         "selector": { }
      }
   },
   "walm-deployment.json": {
      "apiVersion": "extensions/v1beta1",
      "kind": "Deployment",
      "metadata": {
         "annotations": { },
         "labels": {
            "transwarp.alias": "walm_0000",
            "transwarp.install": "1160",
            "transwarp.name": "walm-srv"
         },
         "name": "walm-srv-1160"
      },
      "spec": {
         "replicas": 1,
         "selector": {
            "matchLabels": {
               "transwarp.install": "1160",
               "transwarp.name": "walm-srv"
            }
         },
         "template": {
            "metadata": {
               "annotations": {
                  "cni.networks": "overlay",
                  "podAntiAffinity": {
                     "requiredDuringSchedulingIgnoredDuringExecution": [
                        {
                           "labelSelector": {
                              "matchLabels": {
                                 "transwarp.alias": "walm_0000",
                                 "transwarp.install": "1160",
                                 "transwarp.name": "walm-srv"
                              }
                           },
                           "namespaces": [
                              "u5003"
                           ],
                           "topologyKey": "kubernetes.io/hostname"
                        }
                     ]
                  },
                  "transwarp.app": "walm"
               },
               "labels": {
                  "transwarp.alias": "walm_0000",
                  "transwarp.install": "1160",
                  "transwarp.name": "walm-srv"
               }
            },
            "spec": {
               "containers": [
                  {
                     "args": [
                        "walm serv"
                     ],
                     "env": [
                        {
                           "name": "WALM_HTTP_PORT",
                           "value": 8000
                        },
                        {
                           "name": "WALM_DEBUG",
                           "value": false
                        },
                        {
                           "name": "WALM_DB_NAME",
                           "value": "walm"
                        },
                        {
                           "name": "WALM_DB_TYPE",
                           "value": "mysql"
                        },
                        {
                           "name": "WALM_TABLE_PREFIX",
                           "value": ""
                        },
                        {
                           "name": "HTTP_READ_TIMEOUT",
                           "value": 0
                        },
                        {
                           "name": "HTTP_READ_TIMEOUT",
                           "value": 0
                        },
                        {
                           "name": "ZIPKIN_URL",
                           "value": ""
                        },
                        {
                           "name": "TILLER_CONN_TIMEOUT",
                           "value": 0
                        },
                        {
                           "name": "KUBE_CONTEXT",
                           "value": "default"
                        },
                        {
                           "name": "WALM_OAUTH",
                           "value": false
                        },
                        {
                           "name": "WALM_JWTSECRET",
                           "value": ""
                        }
                     ],
                     "image": "172.16.1.99:5000/gold/walm:tdc-1.0",
                     "imagePullPolicy": "Always",
                     "name": "walm-srv",
                     "resources": {
                        "limits": {
                           "cpu": "4",
                           "memory": "4Gi"
                        },
                        "requests": {
                           "cpu": "1",
                           "memory": "2Gi"
                        }
                     }
                  }
               ],
               "initContainers": [
                  {
                     "command": [
                        "walm migrate"
                     ],
                     "env": [
                        {
                           "name": "WALM_HTTP_PORT",
                           "value": 8000
                        },
                        {
                           "name": "WALM_DEBUG",
                           "value": false
                        },
                        {
                           "name": "WALM_DB_NAME",
                           "value": "walm"
                        },
                        {
                           "name": "WALM_DB_TYPE",
                           "value": "mysql"
                        },
                        {
                           "name": "WALM_TABLE_PREFIX",
                           "value": ""
                        },
                        {
                           "name": "HTTP_READ_TIMEOUT",
                           "value": 0
                        },
                        {
                           "name": "HTTP_READ_TIMEOUT",
                           "value": 0
                        },
                        {
                           "name": "ZIPKIN_URL",
                           "value": ""
                        },
                        {
                           "name": "TILLER_CONN_TIMEOUT",
                           "value": 0
                        },
                        {
                           "name": "KUBE_CONTEXT",
                           "value": "default"
                        },
                        {
                           "name": "WALM_OAUTH",
                           "value": false
                        },
                        {
                           "name": "WALM_JWTSECRET",
                           "value": ""
                        }
                     ],
                     "image": "172.16.1.99:5000/gold/walm:tdc-1.0",
                     "imagePullPolicy": "Always",
                     "name": "walm-init"
                  }
               ],
               "priority": 0,
               "terminationGracePeriodSeconds": 0
            }
         }
      }
   },
   "walm-hl-svc.json": {
      "apiVersion": "v1",
      "kind": "Service",
      "metadata": {
         "annotations": {
            "service.alpha.kubernetes.io/tolerate-unready-endpoints": "true"
         },
         "labels": {
            "k8s-app": "walm-srv-hl",
            "kubernetes.io/headless-service": "true",
            "transwarp.alias": "walm_0000",
            "transwarp.install": "1160",
            "transwarp.name": "walm-srv-hl"
         },
         "name": "walm-srv-hl-1160"
      },
      "spec": {
         "clusterIP": "None",
         "ports": [
            {
               "name": "walm-port",
               "port": 8000,
               "protocol": "TCP"
            }
         ],
         "selector": {
            "transwarp.install": "1160",
            "transwarp.name": "walm-srv"
         }
      }
   },
   "walm-svc.json": {
      "apiVersion": "v1",
      "kind": "Service",
      "metadata": {
         "annotations": { },
         "labels": {
            "k8s-app": "walm-srv",
            "transwarp.alias": "walm_0000",
            "transwarp.install": "1160",
            "transwarp.name": "walm-srv"
         },
         "name": "walm-srv-1160"
      },
      "spec": {
         "ports": [
            {
               "name": "port",
               "port": 8000,
               "protocol": "TCP",
               "targetPort": 8000
            }
         ],
         "selector": {
            "transwarp.install": "1160",
            "transwarp.name": "walm-srv"
         }
      }
   }
}
